app/models/csv_export.rb

```diff
@@ -0,0 +1,57 @@
+class CsvExport < ActiveRecord::Base
+  belongs_to :user
+
```

Comment:
list of possibly status values as constant and then validate that status is in that list